examples/controlnet/train_controlnet_sdxl.py

```diff
@@ -899,6 +899,7 @@ def load_model_hook(models, input_dir):
 
     if args.gradient_checkpointing:
         controlnet.enable_gradient_checkpointing()
+        unet.enable_gradient_checkpointing()
```

Comment:
some gradients also flow through the unet so it's always safer IMO to enable it as well